runtime/README.md

```diff
@@ -711,8 +711,83 @@ Command Code: `0x434B_4558` ("CKEX")
 | fips\_status        | u32       | Indicates if the command is FIPS approved or an error.
 | certify\_key\_resp  | u8[2176]  | Certify Key Response.
 
-## Checksum
+### SET\_AUTH\_MANIFEST
```

Comment:
Is there a document section we can link to here with the manifest format and general design?

Reply:
We have the design document on the shared drive; will that suffice, or should we update the Runtime README.md? https://docs.google.com/document/d/1BofLEAzeszgAGVcaEjTxM7tDxwIq9KhVXuqGUlrg1sc/edit#heading=h.z8ehugjyw5jp

Reply:
That's fine for now. We can move it to markdown later.